homework-submission/week1/src/components/TodoList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import TodoItem from './TodoItem';
+import './TodoList.css';
+import { observer, inject } from 'mobx-react';
+
+@inject('todos')
+@observer
+class TodoList extends React.Component {
+  render() {
+    const { todos, state } = this.props.todos;
+    const list = todos.map((item, i) => <TodoItem key={i} todo={item} />);
```

Comment:
you have an `item._id` for the key, using the index may behave un-expectedly, which means you have a bugs expects 💔 .